### PR TITLE
Compatibility doc: Allow Unicode in URIs that resemble UNC shares

### PR DESCRIPTION
**Pull request by rmkerr**
This additional page documents a change to `System.Uri` that fixes parsing for UNC shares when Unicode is present. This is the last of our 4.7.2 breaking changes to URI, and I think it is the least likely to be a real problem. Again though, we're taking a cautious approach with URI :)

cc: @karelz @davidsh @rpetrusha

### CODE REVIEW

File: Documentation/compatibility/uri-unc-shares-normalized.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Edge
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+In <xref:System.Uri?displayProperty=fullName>, constructing a file URI containing both a UNC share name and Unicode characters will no longer result in a URI with invalid internal state. The behavior will change only when all of the following are true:
```
Comment (rpetrusha) [inline Documentation/compatibility/uri-unc-shares-normalized.md]: will no longer result --> no longer results
a URI --> an URI ?

File: Documentation/compatibility/uri-unc-shares-normalized.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - The URI has the scheme `file:`, and is followed by four or more slashes.
```
Comment (rpetrusha) [inline Documentation/compatibility/uri-unc-shares-normalized.md]: no comma after "file"

File: Documentation/compatibility/uri-unc-shares-normalized.md (hunk 1 above)
Comment (rpetrusha) [reply Documentation/compatibility/uri-unc-shares-normalized.md]: I have no idea of what I was thinking, @rmkerr! It obviously should be "a URI".

**Comment by rpetrusha** [comment]
I'll merge your PR now, @rmkerr.

**Comment by rmkerr** [comment]
Great! Thanks for helping me get all of these changes in so quickly.

**Pull request merged**